### PR TITLE
chore(DEV-18293): [Stats] Fix hover issue

### PR DESCRIPTION
**Pull request by rociobaena**
Fix hover issue:
image

## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

File: packages/cascara/src/atoms/Stat/Stat.module.scss [hunk 1]
```diff
@@ -29,6 +29,8 @@ $widget-background: #f0f0f0;
 
   &.clickable {
     position: relative;
+    transform: translate3d(0, 0, 0);
+    -webkit-transform: translate3d(0, 0, 0);
```
Comment (memecgavante) [inline packages/cascara/src/atoms/Stat/Stat.module.scss:34]: 👍

**Comment by rociobaena** [comment]
> Is the browser testing 📸 taken from safari ? @rociobaena

Yes it is.

**Review by memecgavante — COMMENTED** [review]
This Looks good to me @rociobaena ! ✨  But I have a question, was the browser testing SS 📸 taken in Safari ?

**Review by memecgavante — APPROVED** [review]
Awesome! then, approved :)

**Review by lordpixel — APPROVED** [review]

**Pull request merged**